cranelift-module/src/module.rs

```diff
@@ -648,6 +648,9 @@ where
     ///
     /// Use `get_finalized_function` and `get_finalized_data` to obtain the final
     /// artifacts.
+    ///
+    /// This method is not relevant for `Backend` implementations that do not provide
+    /// finalized functions and data objects.
```

Comment:
What if I say it's only relevant for JIT backends?

Reply:
That's a bit more prescriptive than I prefer for documentation, but I won't push back on it for lack of counterexample.

Reply:
Ok, so instead of using ambiguous English, let's refer to the condition in the code instead. I've moved this change to a new commit that also updates the comments on other functions.

Reply:
Yeah I think that's a good call.